### ISSUE TITLE
cdbc6421 Version broken in Automatic1111 Version 1.5.2 

### ISSUE THREAD

**Issue by LVNSDCAPE** [thygate/stable-diffusion-webui-depthmap-script#336]
Version from 2 days ago 
https://github.com/thygate/stable-diffusion-webui-depthmap-script/commit/cdbc642117a584b55a8929a59cfc838b75f34f39

has entire broken the functionality in Automatic 1111 1.5.2  

After updating in Extention and restarting the application ( closing A1111 ) 
right from the start it displays errors in cmd console 

*** Error loading script: depthmap.py
    Traceback (most recent call last):
      File "L:\SD\modules\scripts.py", line 319, in load_scripts
        script_module = script_loading.load_module(scriptfile.path)
      File "L:\SD\modules\script_loading.py", line 10, in load_module
        module_spec.loader.exec_module(module)
      File "<frozen importlib._bootstrap_external>", line 883, in exec_module
      File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
      File "L:\SD\extensions\stable-diffusion-webui-depthmap-script\scripts\depthmap.py", line 8, in <module>
        from src import common_ui
      File "L:\SD\extensions\stable-diffusion-webui-depthmap-script\src\common_ui.py", line 6, in <module>
        from src import backbone, video_mode
      File "L:\SD\extensions\stable-diffusion-webui-depthmap-script\src\video_mode.py", line 8, in <module>
        from src import core
      File "L:\SD\extensions\stable-diffusion-webui-depthmap-script\src\core.py", line 26, in <module>
        from src.depthmap_generation import ModelHolder
      File "L:\SD\extensions\stable-diffusion-webui-depthmap-script\src\depthmap_generation.py", line 21, in <module>
        from lib.multi_depth_model_woauxi import RelDepthModel
    ModuleNotFoundError: No module named 'lib.multi_depth_model_woauxi'

After launching there is nothing inside Automatic1111 in scripts dropdown but extention is still checkboxed as installed 

I also deleted the depthmap script and reinstalled which ended in exact same behaviors then.

**Comment by semjon00** [comment]
Please upgrade?

**Comment by LVNSDCAPE** [comment]
Ugrade to Automatic1111 
1.6x ?  
If you mean this no thx.

It is extreme slow in 1.5 generation , infested with bugs, does no longer reproduce png pictures due to sampler changes , has horrible gui changes. There is so many negative currently in Automatic1111 1.6x  which makes no sense at all to upgrade to this. 

If you have no plans for a update or fix since your new version obvious broke it, can people downgrade to the pre version of your plugin/extention?

**Comment by semjon00** [comment]
Ok, this is understandable. I will try to fix it, but no guaranteef timeframe. Uni just started for me and I am having it hard.

**Comment by LVNSDCAPE** [comment]
Hi again.
I noticed that i installed this new extention today 
https://github.com/Nekos4Lyfe/TokenMixer

which has been recommended here in a reply by the programmer AdComfortable1544
https://www.reddit.com/user/AdComfortable1544/ in this reddit thread discussion 

https://www.reddit.com/r/civitai/comments/16fs7qh/comment/k03wdv8/?utm_source=share&utm_medium=web2x&context=3

and it is causing this issue.

If i deactivate it your extention works like a charm again.

I noticed that since i opened a backup version Automtic1111 
1.5.2 which included a working depthmap extention and then installed the Tokenmixer 
again and same issue happened.
